assets/xml/objects/object_dnj.xml

```diff
@@ -1,86 +1,96 @@
+        <DList Name="gDekuButlerSnoutDL" Offset="0xC248" />
+        <DList Name="gDekuButlerRightMustacheCurlDL" Offset="0xC370" />
+        <DList Name="gDekuButlerRightMustacheBaseDL" Offset="0xC450" />
+        <DList Name="gDekuButlerLeftMustacheCurlDL" Offset="0xC4E0" />
+        <DList Name="gDekuButlerLeftMustacheBaseDL" Offset="0xC5C0" />
+        <DList Name="gDekuButlerRightForearmDL" Offset="0xC650" />
+        <DList Name="gDekuButlerRightHandDL" Offset="0xC7C0" />
+        <DList Name="gDekuButlerParasolUpperPetalsDL" Offset="0xC8D8" />
+        <DList Name="gDekuButlerParasolMiddlePetalsDL" Offset="0xC988" />
+        <DList Name="gDekuButlerParasolLowerPetalsDL" Offset="0xCA38" />
+        <DList Name="gDekuButlerParasolHandleDL" Offset="0xCAE8" />
+        <DList Name="gDekuButlerRightUpperArmDL" Offset="0xCBE8" />
+        <DList Name="gDekuButlerPelvisDL" Offset="0xCC90" />
+        <DList Name="gDekuButlerLeftShinDL" Offset="0xCD48" />
+        <DList Name="gDekuButlerLeftFootDL" Offset="0xCE00" />
+        <DList Name="gDekuButlerLeftThighDL" Offset="0xCEC8" />
+        <DList Name="gDekuButlerRightShinDL" Offset="0xCF88" />
+        <DList Name="gDekuButlerRightFootDL" Offset="0xD040" />
+        <DList Name="gDekuButlerRightThighDL" Offset="0xD108" />
+
+        <!-- Deku Butler Textures -->
+        <Texture Name="gDekuButlerEyeTex" OutName="deku_butler_eye" Format="rgba16" Width="16" Height="16" Offset="0xD1C8" />
+        <Texture Name="gDekuButlerBodyAndCandleHolderTex" OutName="deku_butler_body_and_candle_holder" Format="rgba16" Width="16" Height="16" Offset="0xD3C8" />
+        <Texture Name="gDekuButlerMouthTex" OutName="deku_butler_mouth" Format="rgba16" Width="8" Height="8" Offset="0xD5C8" />
+        <Texture Name="gDekuButlerLeafTex" OutName="deku_butler_leaf" Format="rgba16" Width="16" Height="16" Offset="0xD648" />
+        <Texture Name="gDekuButlerParasolPetalTex" OutName="deku_butler_parasol_petal" Format="rgba16" Width="16" Height="32" Offset="0xD848" />
+        <Texture Name="gDekuButlerCandleTex" OutName="deku_butler_candle" Format="rgba16" Width="16" Height="8" Offset="0xDC48" />
+        <Texture Name="gDekuButlerHairTex" OutName="deku_butler_hair" Format="rgba16" Width="16" Height="16" Offset="0xDD48" />
+        <Texture Name="gDekuButlerBowtieCenter" OutName="deku_butler_bowtie_center" Format="rgba16" Width="8" Height="16" Offset="0xDF48" />
+
+        <!-- Deku Butler Limbs -->
+        <Limb Name="gDekuButlerRootLimb" Type="Standard" EnumName="DEKU_BUTLER_LIMB_ROOT" Offset="0xE048" />
+        <Limb Name="gDekuButlerPelvisLimb" Type="Standard" EnumName="DEKU_BUTLER_LIMB_PELVIS" Offset="0xE054" />
+        <Limb Name="gDekuButlerRightThighLimb" Type="Standard" EnumName="DEKU_BUTLER_LIMB_RIGHT_THIGH" Offset="0xE060" />
+        <Limb Name="gDekuButlerRightShinLimb" Type="Standard" EnumName="DEKU_BUTLER_LIMB_RIGHT_SHIN" Offset="0xE06C" />
```

Comment:
I thought we used `LowerLeg` for these (since the shin is only part of the lower half of the leg), but a quick search tells me otherwise.

Reply:
Yeah, I've actually *never* heard that lol. I've only seen `LowerLeg` used for non-humanoid limbs (see, for example, Pp or Grasshopper).